Audio Service/AudioSourceFader.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AudioSourceFader : MonoBehaviour
+{
+    public AudioSource audioSource;
+    private float _fadeDuration = 2f;
```

Comment:
The designer should be able to modify this in inspector so use `[SerializeField]`